MATLAB/Demos/FDK_half.m

```diff
@@ -0,0 +1,192 @@
+function [res]=FDK_half(proj,geo,angles,varargin)

```

Comment:
I think its better to have everything under the same function `FDK()`

Reply:
Yeah, I can merge it into FDK. But my main concern is, probably that will make the code in FDK hard to read, which is not friendly to TIGRE users, especially new ones that are no familiar with CT recon.

Reply:
Hum, I still think its better suited for TIGRE to have it in the same function. All algorithms are unique, and ultimately this is just a weighting factor to FDK, but still FDK.  Ideally we would have a check if the detector is displaced, and a comment explaining why this is required, or simply a link to the paper.

Reply:
OK. I'm fine to merge it into FDK. But I'm kind of busy recently, it will take a while to finish the work. If you prefer doing the job to keep uniform code style, I can explain all the details of _FDK_half_ to facilitate your recoding.